README.md

```diff
@@ -59,7 +59,7 @@ This project is a Flask-based web application that allows users to manage and ed
 
 6. Make the tunnel, connecting the database
     ```
-    ssh -N -v <your_shell_username>@dev.toolforge.org -L localhost:3306:enwiki.analytics.db.svc.eqiad.wmflabs:3306
+    ssh -N -v sarthakparashar@dev.toolforge.org -L localhost:3306:enwiki.analytics.db.svc.eqiad.wmflabs:3306
```

Comment:
I don't see a reason to edit this file at all as well.

Reply:
I'm sorry I will change that issue